VogtPayroll5/Payroll.cs

```diff
@@ -1,34 +1,57 @@
 ﻿using System;
 using System.Collections.Generic;
 using System.Text;
+using System.IO;
 
 namespace VogtPayroll5
 {
     class Payroll
     {
+        //total number of hours worked by all employees, and total gross pay as well
+        public int countHours;
+        public static decimal totalGrossPay;
```

Comment:
Get rid of these from being class variables. Instead, create a local variable to temporarily hold the value in the GetTotalHoursWorked function